go/host/hostrunner/cli.go

```diff
@@ -51,7 +52,8 @@ func ParseConfig() config.HostConfig {
 	clientRPCTimeoutSecs := flag.Uint64(clientRPCTimeoutSecsName, uint64(cfg.ClientRPCTimeout.Seconds()), clientRPCTimeoutSecsUsage)
 	enclaveRPCAddress := flag.String(enclaveRPCAddressName, cfg.EnclaveRPCAddress, enclaveRPCAddressUsage)
 	enclaveRPCTimeoutSecs := flag.Uint64(enclaveRPCTimeoutSecsName, uint64(cfg.EnclaveRPCTimeout.Seconds()), enclaveRPCTimeoutSecsUsage)
-	p2pAddress := flag.String(p2pAddressName, cfg.P2PAddress, p2pAddressUsage)
+	p2pBoundAddress := flag.String(p2pBindAddressName, cfg.P2PBindAddress, p2pBindAddressUsage)
```

Comment:
Minor but worth sticking to `bind` naming instead of `bound`? Easier to search if we're consistent

Reply:
Good point, fixed 👍